packages/dds/sequence/src/intervalCollection.ts

```diff
@@ -735,23 +790,52 @@ export class IntervalCollectionView<TInterval extends ISerializableInterval> ext
         return interval;
     }
 
-    public deleteInterval(serializedInterval: ISerializedInterval, local: boolean, op: ISequencedDocumentMessage) {
-        const interval = this.localCollection.removeInterval(serializedInterval.start, serializedInterval.end);
+    /**
+     * @deprecated - delete(start,end) will be removed in future.  Please use deleteIntervalById instead.
+     */
+    public delete(start: number, end: number) {
+        return this.deleteIntervalById(this.localCollection.createLegacyId(start, end));
```

Comment:
it doesn't actually work this way. you need to actually set the reserved id field in props on load from the snapshot, as in the case of shared inverval on a shared string the endpoints can change over time as modifications happen. 

overall, i think we might be tackling too many things is this pr, which is making it hard to get any of them in.

Reply:
Updating to handle the load case, which is the one I wasn't handling.